### PR TITLE
[Deploy] Add lastUsedBy annotation to the project

### PR DESCRIPTION
**Pull request by avano**
//skip-ci

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/TestSuiteParent.java [hunk 1]
```diff
@@ -41,6 +43,17 @@ public static void beforeTests() {
             TestUtils.sleepIgnoreInterrupt(10 * 1000L);
         }
 
+        // You can't create project with annotations/labels - https://github.com/openshift/origin/issues/3819
+        // So add them after the project is created
+        // @formatter:off
+        Map<String, String> annotations = TestUtils.map("syndesis-qe/lastUsedBy", System.getProperty("user.name"));
+        OpenShiftUtils.getInstance().namespaces().withName(TestConfiguration.openShiftNamespace()).edit()
+            .editMetadata()
+                .addToAnnotations(annotations)
```
Comment (asmigala) [inline utilities/src/main/java/io/syndesis/qe/TestSuiteParent.java]: I'm not 100 % sure about the difference between labels and annotations but I found that labels are for things seen by users and annotations are for internal purposes. Would it make more sense to add this as a label?

Comment (avano) [reply utilities/src/main/java/io/syndesis/qe/TestSuiteParent.java]: that's exactly why I put it in the annotations :grin: , but I can add it to labels, no problem

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**